src/hotspot/cpu/x86/interp_masm_x86.cpp

```diff
@@ -1776,13 +1776,13 @@ void InterpreterMacroAssembler::record_item_in_profile_helper(Register item, Reg
 // Example state machine code for three profile rows:
 //   // main copy of decision tree, rooted at row[1]
 //   if (row[0].rec == rec) { row[0].incr(); goto done; }
-//   if (row[0].rec != NULL) {
+//   if (row[0].rec != null) {
```

Comment:
nullptr

Reply:
Missed change to nullptr as you suggested.